### PR TITLE
"Developing the right culture"  by Rob Johnson

### PR DESCRIPTION
**Pull request by jodiedoubleday**
### Blog - "Developing the right culture"
A blog post originally created on Medium which we now have permission to move onto Hungry Geek
**PR on behalf of @notABluesSinger**
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by lukefrake** [comment]
I'll take 1 review on this

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## So, how have we got here?
+
+Again, this is quite a difficult question to answer, but that’s the point of this, so I’ll try.
+
+I think it is paramount to have people in your team who care. Obviously, not just about themselves. They need to care about the product, the customer, their colleagues and about themselves. Everyones well-being should be a consideration.
+
+We all have those moments when our verbal filter malfunctions, but we can’t all be like that, all of the time. But, we *should* be like that some of the time, allow yourselves to be brutally honest with each other, as long as it is constructive.
+
+Management **MUST **trust every member of each team to be able to do their job and to do it well. You have to have that relationship where you know you’re allowed to get on and produce the best that you’re capable of and, in turn, your manager should be rewarded for that trust, by you delivering the best that you can.
```
Comment (lukefrake) [inline _posts/2015-02-16-developing-the-right-culture.md]: Should be **MUST**

**Comment by lukefrake** [comment]
+1 once typo is corrected

**Comment by hpoom** [comment]
What's the blog plan now @jodiedoubleday @lukefrake 

Are we moving all Airport Cafe posts to Hungry Geek? Or are we waiting till we have a tech brand and moving both into that?

**Comment by jodiedoubleday** [comment]
probably not the best place to have a conversation about this... i'll contact you on slack.

**Comment by dancork** [comment]
I'll take editor 2

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+author: Rob Johnson
+tags: culture
+comments: true
+---
+
+Recently, at work, we’ve been discussing how we have managed to create the positive, engaging culture that we all feel we have.
+
+We’ve spoken to people outside of our company and found that they often struggle trying to create the kind of culture we see when we look at some of the [Times Best 100 Companies to work for](http://www.thesundaytimes.co.uk/best100companies) and want to know how you can create the kind of culture you can be proud of.
+
+And it has been extremely hard to pinpoint exactly what it is that has created the environment that we all spend our days bouncing around in.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: Shouldn't start sentence with a conjunction.

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+* We know our weaknesses as well as our strengths.
+
+* We have amazing play time.
+
+* We, genuinely, are all friends and have each others backs.
+
+## So, how have we got here?
+
+Again, this is quite a difficult question to answer, but that’s the point of this, so I’ll try.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: Last comma is not needed `but that’s the point of this so I’ll try`.

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+I think it is paramount to have people in your team who care. Obviously, not just about themselves. They need to care about the product, the customer, their colleagues and about themselves. Everyones well-being should be a consideration.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: `to care about the product, the customer, their colleagues and about themselves` saying about twice.

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+We all have those moments when our verbal filter malfunctions, but we can’t all be like that, all of the time. But, we *should* be like that some of the time, allow yourselves to be brutally honest with each other, as long as it is constructive.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: Last comma is not needed here `but we can’t all be like that all of the time`

Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: Not great to start a sentence with `But`. Maybe break this into a couple of sentences too. Something like...

`However, we *should* be like that some of the time. Allow yourselves to be brutally honest with each other, as long as it is constructive.

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+Don’t let people be afraid to question decisions, that should be expected. There is no point having a room full of "yes men".
+
+Also, make it common place to ask for help. If you’re working on something that you know your colleague knows better than you, get their help, their input and produce better work because of the collaboration.
+
+Our people are like this.
+
+## How can other companies build their own culture?
+
+It’s very hard to change an existing culture, but I think it is so much easier coming from the top down. If you’re in a senior position, use it to your advantage. Let your juniors challenge you, be open to their ideas and honest with your responses.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: `If you’re in a senior position, use it to your advantage.` no need for the comma.

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+Talk to your colleagues, be the one to ask the questions, be the one to ask for help, while explaining that you think you’d get a better end product and learn at the same time with that help.
+
+Be the organiser. It’s easy, start with something everybody loves…food. Create a monthly "payday takeaway". **Come on**, that sounds good, right?
+
+Do research and find examples and share them with your team and your seniors. Show that it makes a difference.
+
+## In closing…
+
+The funny thing is, that good people will create a good culture, which will attract good people, which will create an even better culture. It’s almost like a self learning, self improving system. As long as you keep it clean, don’t let the bad stuff build up and don’t be afraid to have a bit of a clean out to stop any bad "juju" spreading.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: Comma in the wrong place. Needs to be after `that`.

`The funny thing is that, good people will create a good culture`

File: _posts/2015-02-16-developing-the-right-culture.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+We all have to work, but the really good people enjoy their jobs and they pick jobs that they will be able to enjoy or be able to make a difference and make more enjoyable, if those good people work for you, *well done*. That is because you cared enough to make their job enjoyable.
```
Comment (dancork) [inline _posts/2015-02-16-developing-the-right-culture.md]: Seems like a long sentence. Could turn the comma here `make more enjoyable, if those good people work` in to a full stop.

**Comment by dancork** [comment]
Good post Rob. Just a few grammatical changes/suggestions.

**Comment by dancork** [comment]
:+1:

**Pull request merged**